content/blog/lintr-3-0-0/index.md

```diff
@@ -0,0 +1,229 @@
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We are very excited to announce the release of [lintr](https://lintr.r-lib.org) 3.0.0! lintr provides
+both a framework for [static analysis](https://www.perforce.com/blog/sca/what-static-analysis) of R packages
+and scripts and a variety of linters, e.g. to enforce the [tidyverse style guide](https://style.tidyverse.org/).
+
+You can install it from CRAN with:
+
+```r
+install.packages("lintr")
+```
+
+Check our vignettes for a quick introduction to the package:
+ - Getting started ([`vignette("lintr")`](https://lintr.r-lib.org/articles/lintr.html))
+ - Integrating lintr with your preferred IDE ([`vignette("editors")`](https://lintr.r-lib.org/articles/editors.html))
+ - Integrating lintr with your preferred CI tools ([`vignette("continuous-integration")`](https://lintr.r-lib.org/articles/continuous-integration.html))
+
+We've also added `lintr::use_lintr()` for a usethis-inspired interactive tool to configure lintr for your package/repo.
+
+This blog post will highlight the biggest changes coming in this update which drove us to declare it a major release.
+
+# Release highlights
+
+## Selective exclusions
+
+lintr now supports targeted exclusions of specific linters through an extension of the `# nolint` syntax.
+
+Consider the following example:
+
+```r
+T_and_F_symbol_linter=function(){
+  list()
+}
+```
+
+This snippet generates 5 lints:
+
+ 1. `object_name_linter()` because the uppercase `T` and `F` do not match `lower_snake_case`
+ 2. `brace_linter()` because `{` should be separated from `)` by a space
+ 3. `paren_body_linter()` because `)` should be separated from the function body (starting at `{`) by a space
+ 4. `infix_spaces_linter()` because `=` should be surrounded by spaces on both sides
+ 5. `assignment_linter()` because `<-` should be used for assignment
+
+The first lint is spurious because `t` and `f` do not correctly convey that this linter targets
+the symbols `T` and `F`, so we want to ignore it. Prior to this release, we would have to
+"throw the baby out with the bathwater" by suppressing _all five lints_ like so:
+
+```r
+T_and_F_symbol_linter=function(){ # nolint. T and F are OK here.
+  list()
+}
+```
+
+This hides the other four lints and prevents any new lints from being detected on this line in the future,
+which on average allows the overall quality of your projects/scripts to dip.
+
+With the new feature, you'd write the exclusion like this instead:
+
+```r
+T_and_F_symbol_linter=function(){ # nolint: object_name_linter. T and F are OK here.
+  list()
+}
+```
+
+By qualifying the exclusion, the other 4 lints will be detected and exposed by `lint()` so
+that you can fix them! See `?exclude` for more details.
+
+## Linter factories
+
+As of lintr 3.0.0, _all_ linters must be [function factories](https://adv-r.hadley.nz/function-factories.html).
+
+Previously, only parameterizable linters (such as `line_length_linter`, which takes a parameter controlling how
```

Comment:
```suggestion
Previously, only parameterizable linters (such as `line_length_linter()`, which takes a parameter controlling how
```

Reply:
House style is to use parentheses for all functions, and that way downlit will also automatically link to the docs.